src/plugins/vis_type_xy/public/utils/use_color_picker.tsx

```diff
@@ -61,18 +61,25 @@ export const useColorPicker = (
         onClose();
       };
 
+      // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Could you leave a comment explaining why we need to break this rule here? We do this in a few places in Lens but it's helpful to note why.

Reply:
Sure, I noticed the rule has a hard time knowing when it's in a component, hook, or otherwise. Have you noticed this too?

Reply:
4fb7b62